config/locales/server.en.yml

```diff
@@ -5534,6 +5535,7 @@ en:
         other: "Automatically revoked, created at more than %{count} days ago"
       revoked: Revoked
       restored: Restored
+    bulk_user_delete: "deleted in a bulk delete operation"
```

Comment:
Is "deleted" intentionally all lowercase here?

Reply:
Will answer for him here, but yes this is the format of all our staff action logs...it's another remnant of a lot of the Discourse UI being lowercase in the past